easybuild/easyconfigs/s/Siesta/Siesta-4.0-intel-2016.02-GCC-4.9.eb

```diff
@@ -0,0 +1,16 @@
+name = 'Siesta'
+version = '4.0'
+
+homepage = 'http://departments.icmab.es/leem/siesta'
+description = """SIESTA is both a method and its computer program implementation, to perform efficient electronic
+ structure calculations and ab initio molecular dynamics simulations of molecules and solids."""
+
+toolchain = {'name': 'intel', 'version': '2016.02-GCC-4.9'}
+toolchainopts = {'usempi': True}
+
+sources = [SOURCELOWER_TGZ]
+source_urls = ['https://launchpad.net/%(namelower)s/%(version_major_minor)s/%(version)s/+download/']
+
+parallel = 1
```

Comment:
@migueldiascosta is this still required? if so, it's probably better to tackle this via the easyblock?